django/control/urls.py

```diff
@@ -4,8 +4,8 @@
 from control.api.router import router
 
 urlpatterns = [
-    path("", include(router.urls)),
-    path("preview/", views.PreviewEndpoint.as_view(), name="preview"),
-    path("delete-resources/", views.ResourceEndpoint.as_view(), name="delete-resources"),
-    path("scripts/", views.ScriptsEndpoint.as_view(), name="scripts"),
+    path("api/control/", include(router.urls)),
```

Comment:
why prefixing every route with `api` ? Shouldn't be the role of the reverse proxy to eventually manage that?

Reply:
1. Some routes be associated with endpoints that are not part of the REST api (for instance, `admin` routes. Or we could define custom views)
2. Using nginx for routing creates a hard dependency, means you could not develop the frontend without bringing nginx into the mix. The current way is simpler

Reply:
About the second point, I don't think that's really an issue. With CRA we define different api root urls depending on the environment anyway. I don't know if the api should be aware that it will be hosted on the same domain or from where the client will be served for example.